macros/optimize_tables.sql

```diff
@@ -99,6 +99,10 @@ OPTIMIZE archipelago_ethereum.events;
 OPTIMIZE labels.all;
 {% endset %}
 
+{% set cow_protocol_ethereum_table_optimizer %}
+OPTIMIZE cow_protocol_ethereum.trades;
+{% endset %}
+
```

Comment:
Similar to the comment above, I have named this in a way that suggests other tables in the same namespace be optimized within this block. Not sure if this considered appropriate by the Dune team.